curvefs/src/client/rpcclient/mds_client.cpp

```diff
@@ -141,12 +146,14 @@ FSStatusCode MdsClientImpl::MountFs(const std::string& fsName,
     return ReturnError(rpcexcutor_.DoRPCTask(task, mdsOpt_.mdsMaxRetryMS));
 }
 
-FSStatusCode MdsClientImpl::UmountFs(const std::string& fsName,
-                                     const std::string& mountPt) {
+FSStatusCode MdsClientImpl::UmountFs(const std::string &fsName,
+                                     const std::string &mountPt) {
     auto task = RPCTask {
+        mdsClientMetric_.umountFs.qps.count << 1;
         UmountFsResponse response;
         mdsbasecli_->UmountFs(fsName, mountPt, &response, cntl, channel);
         if (cntl->Failed()) {
+            mdsClientMetric_.umountFs.eps.count << 1;
```

Comment:
Should we cout eps when ret != FSStatusCode::OK like MountFs？

Reply:
this is rpc error, FSStatsCode not ok is logic.